src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1 +1,15 @@
-export const CommentInfo = () => <>Put the comment here</>;
+export const CommentInfo = ({ name, email, body }) => (
```

Comment:
```suggestion
export const CommentInfo = (comment: { name, email, body }) => (
```

Reply:
Sorry to say, but suggested code doesn't fit current needs as `npm start` refuse to build it. Reverted to the original  solution.